frontend/src/scenes/notebooks/Notebook/Editor.tsx

```diff
@@ -65,6 +65,7 @@ export function Editor({
             CustomDocument,
             StarterKit.configure({
                 document: false,
+                gapcursor: false,
```

Comment:
I don't think this is mergeable. I find the weird gap cursor preferable to not being able to place a cursor between two nodes...

Not against this change, as long as we do the extra work to make sure you can add a line between nodes when we click there. Might be best done by detecting the selection and then inserting a line if there isn't one (not sure how hacky that would be)